common/scripts/mesh-helper.sh

```diff
@@ -43,21 +43,34 @@ EOF
   fi
 }
 
-generate_br_lan_ip() {
+generate_bridge_ip() {
   local mesh_if_mac
-  mesh_if_mac="$(ip -brief link | grep "$MESH_VIF" | awk '{print $3; exit}')"
+
+  bridge_name=$(echo "$BRIDGE" | cut -d' ' -f1)
+
+  mesh_if_mac="$(ip -brief link | grep "${id0_MESH_VIF}" | awk '{print $3; exit}')"
```

Comment:
What is $id0_MESH_VIF? If it is an interface name (like "wlan0" as opposed to an index like "1") then it is much faster and more reliable to `MESH_IF_MAC=$(cat /sys/class/net/$id0_MESH_VIF/address)` :)

Reply:
fixed in https://github.com/tiiuae/mesh_com/commit/b67d5f9e3c84fcec3732cb04a35297e5acbe08f4